packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -163,59 +191,58 @@ class _AddNewProductPageState extends State<AddNewProductPage>
     }
 
     return WillPopScope(
-      onWillPop: () async {
-        if (_isPopulated) {
-          return true;
-        }
-        final bool? leaveThePage = await showDialog<bool>(
-          context: context,
-          builder: (final BuildContext context) => SmoothAlertDialog(
-            title: appLocalizations.new_product,
-            actionsAxis: Axis.vertical,
-            body: Text(appLocalizations.new_product_leave_message),
-            positiveAction: SmoothActionButton(
-              text: appLocalizations.yes,
-              onPressed: () => Navigator.of(context).pop(true),
-            ),
-            negativeAction: SmoothActionButton(
-              text: appLocalizations.cancel,
-              onPressed: () => Navigator.of(context).pop(false),
-            ),
-          ),
-        );
-        if (leaveThePage == true) {
-          AnalyticsHelper.trackEvent(
-            widget.events[EditProductAction.leaveEmpty]!,
-            barcode: barcode,
-          );
-        }
-        return leaveThePage ?? false;
-      },
+      onWillPop: _onWillPop,
       child: SmoothScaffold(
-        appBar: SmoothAppBar(
+        /*appBar: SmoothAppBar(
           title: ListTile(
             title: Text(
               upToDateProduct.productName ?? appLocalizations.new_product,
             ),
             subtitle: Text(barcode),
           ),
-        ),
-        body: Padding(
-          padding: const EdgeInsetsDirectional.symmetric(
-            vertical: VERY_LARGE_SPACE,
-          ),
-          child: SingleChildScrollView(
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: <Widget>[
-                if (widget.displayPictures) _buildCard(_getImageRows(context)),
-                _buildCard(_getNutriscoreRows(context)),
-                _buildCard(_getEcoscoreRows(context)),
-                _buildCard(_getNovaRows(context)),
-                if (widget.displayMisc) _buildCard(_getMiscRows(context)),
-                const SizedBox(height: MINIMUM_TOUCH_SIZE),
-              ],
-            ),
+        ),*/
+        body: SafeArea(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              /*if (widget.displayPictures) _buildCard(_getImageRows(context)),
```

Comment:
Same question as before, why do you comment this code?

Reply:
Oh Sorry, I will remove that comment.
That commented part is used from the Line Number : 230